app/serializers/api/admin/order_serializer.rb

```diff
@@ -16,6 +16,12 @@ def distributor_name
     object.distributor.andand.name
   end
 
+  def display_outstanding_balance
+    return "" if object.outstanding_balance.zero?
```

Comment:
Looking at this now, why not just return 0 and adapt the angular code with != 0?

Reply:
Hah.... you're right I could do this, leave the view logic to the front end again. I'll try and get onto this tomorrow and the cleanup the commits. 

Thanks for the help @luisramos0

Reply:
I was just thinking about this some more. We're using `outstanding_balance` to figure out if the balance is zero but we're sending a different value, `display_outstanding_balance.to_s` in the API response, which will be "$0.00" or whatever the currency symbol is and the dollar amount, which might make this small change tricker than I thought if we need to check if outstanding_balance == <currency_symbol>0.00 .  

I think this is where it would be more useful if the front end formatted the amount in the correct locale instead of it coming from the back end, but that might be a task for another time.

Reply:
ah, yes, I agree. Let's keep it as is it :+1: